sample/bad_form.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
These aren't actually forms, but responses.  Maybe the filenames should be `bad_form_response.json`?

Reply:
All files in `/sample` directory are samples of the API responses. Probably we should rename the directory to `/sample-api-responses`.